dist/variables/ds6/icon-variables.less

```diff
@@ -167,12 +171,18 @@
 @icon-profile-height: 24px;
 @icon-clock-width: 22px;
 @icon-clock-height: 22px;
-@icon-radio-checked-width: 24px;
-@icon-radio-checked-height: 24px;
-@icon-radio-checked-small-width: 16px;
-@icon-radio-checked-small-height: 16px;
-@icon-radio-unchecked-width: 24px;
-@icon-radio-unchecked-height: 24px;
+@icon-radio-checked-width: 18px;
+@icon-radio-checked-height: 18px;
+@icon-radio-checked-large-width: 24px;
+@icon-radio-checked-large-height: 24px;
+@icon-radio-checked-small-width: 18px;
+@icon-radio-checked-small-height: 18px;
+@icon-radio-unchecked-width: 18px;
+@icon-radio-unchecked-height: 18px;
+@icon-radio-unchecked-large-width: 24px;
+@icon-radio-unchecked-large-height: 24px;
+@icon-radio-unchecked-small-width: 18px;
+@icon-radio-unchecked-small-height: 18px;
 @icon-radio-unchecked-small-width: 18px;
 @icon-radio-unchecked-small-height: 18px;
```

Comment:
If unchecked and checked variants have same heights and widths why create separate variables? We can always add new variables if one of them needs a different value in future.

Reply:
Yes, I will use aliases instead. Thanks.

Reply:
Oh, I thought this was the same comment as Andrew's. I see now what you mean. We have it this way mainly because of the mixin that depends on receiving both a width and a height variable for every icon. I know it's a little redundant in this case. We could look into updating the mixin in future, but for now I will leave it as is.